packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.base.tsx

```diff
@@ -156,7 +170,9 @@ export class OverflowSet extends BaseComponent<IOverflowSetProps, {}> implements
 
   private _onRenderItems = (items: IOverflowSetItemProps[]): JSX.Element[] => {
     return items.map((item, i) => {
-      const wrapperDivProps: React.HTMLProps<HTMLDivElement> = { className: css('ms-OverflowSet-item', styles.item) };
+      const wrapperDivProps: React.HTMLProps<HTMLDivElement> = {
+        className: css('ms-OverflowSet-item', this._classNames.item)
```

Comment:
Do we need to keep these literal class names for consumer backwards compatibility?

Reply:
Yeah, we like to keep all original classnames but you can move it to the styles file so the base component has no classes.

Reply:
https://github.com/OfficeDev/office-ui-fabric-react/pull/5172/files#diff-2e6d715dbf4c3c8d6db3a0e95d57913bR17